resources/serverless/charts/webhook/values.yaml

```diff
@@ -38,6 +38,26 @@ pod:
   annotations:
     sidecar.istio.io/inject: "false"
     cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
+  extraProperties:
+    securityContext:
+      runAsUser: 1000 # Optional. Use this setting only when necessary, otherwise delete it. Never set to 0 because this is the ID of root.
+      runAsGroup: 1000 # Optional. Use this setting only when necessary, otherwise delete it. Never set to 0 because this is the ID of root.
+      seccompProfile: # Optional. This option can also be set on container level but it is recommended to set it on Pod level and leave it undefined on container level.
+        type: RuntimeDefault
+      # sysctls: # Optional. Only the following calls are allowed. If it isn't necessary to modify kernel attributes (which usually should be the case), the whole sysctl-section has to be removed.
```

Comment:
Do we need all these comments? maybe better would be to add [a link](https://github.com/kyma-project/community/tree/main/concepts/psp-replacement) to the right docs with a description of which fields should be set and why.  I'm wondering about your opinion.

Reply:
Initially I went with the lazy way of having everything commented right there, to just be able to uncomment whatever is needed. However, come to think of it, these settings will probably never change. So, I will remove all the commented lines, and just link to the docs.